## Red check is a known unrelated flake

The failing `markdownlint` job (which runs full `make check`) is **not** caused by this change. Recorded under Linear CAK-175, which has been reopened.

Failure detail from [the run](https://github.com/ctrl-alt-keith/ai-workflow-playbook/actions/runs/33230827309/job/99043243641):

```
FAIL: test_live_unrelated_worktree_commit_passes_and_records_tolerated_changes
      (test_claude_review_launcher.GovernedClaudeReviewLauncherTests)
AssertionError: 70 != 0
```

That is one of CAK-175's worktree-concurrency fixtures. It reads no documentation, so a prose-only addition to `docs/core-model.md` cannot reach it. Git's background auto-maintenance created `objects/maintenance.lock` during the fixture window; live monitoring classified the new lock as `blocking_lock_change` and emergency-stopped the attempt. The terminal `no_delta_postflight` in the same receipt reports `"passed": true` and reclassifies every affected path as `tolerated` — the design's own final verdict was clean, but the live monitor stopped it first.

Evidence this change is clean, from the same job:

- `markdownlint-cli2`: `Linting: 55 files` / `Summary: 0 issues in 0 files`
- `FAILED (failures=1)` — that single failure, nothing else
- Locally, all five test modules that read `core-model.md` pass 34/34, and `test_markdown_links` passes on both this branch and `main`

Re-running the job should pass unless auto-maintenance fires again. Leaving the check red rather than force-merging, since this PR is a material doctrine candidate that needs governed independent review before promotion regardless.

--- a/Makefile
+++ b/Makefile
@@ -8,10 +8,10 @@ help: ## List available repo-local Makefile targets with short descriptions.
 check: ## Run canonical local validation for local work and CI.
 	@if command -v markdownlint-cli2 >/dev/null 2>&1; then \
 		echo "Running markdownlint-cli2"; \
-		markdownlint-cli2 "**/*.md"; \
+		markdownlint-cli2 "**/*.md" "!.worktrees/**"; \
 	elif command -v markdownlint >/dev/null 2>&1; then \
 		echo "Running markdownlint"; \
-		markdownlint "**/*.md"; \
+		markdownlint --ignore ".worktrees" "**/*.md"; \
 	else \
 		echo "markdownlint is not installed."; \
 		echo "Install markdownlint-cli2 or markdownlint, then rerun 'make check'."; \

--- a/docs/core-model.md
+++ b/docs/core-model.md
@@ -189,22 +189,32 @@ here or in an executor adapter.
 ### Surface Classes
 
 A surface class describes what an execution surface can structurally do,
-independent of vendor or product. Executor adapters map their concrete
-products onto these classes; this document does not enumerate products.
+independent of vendor or product. It states repository locality only. It does
+not state who started the run.
 
 - **Conversational** — no filesystem and no repository locality. Context
   arrives per thread, and any startup contract that depends on reading
   repository files is best-effort rather than guaranteed.
-- **Agentic-local** — real filesystem and repository locality. A startup
-  contract that reads repository and repo-local instruction files genuinely
-  runs.
-- **Agentic-hosted** — agentic execution that may begin with no human present
-  and without guaranteed local filesystem locality.
+- **Agentic-local** — filesystem and repository locality. A startup contract
+  that reads repository and repo-local instruction files genuinely runs.
+- **Agentic-remote** — agentic execution without guaranteed local repository
+  locality. Sources are reached through a connector or another retrieval
+  route rather than through the filesystem.
 
-Surface class constrains which roles a surface can actually perform; it does
-not assign a role, and it does not rank surfaces. Any actor may occupy any
-role its surface class can structurally support. When a required contract
-depends on a capability the current surface class lacks, fail closed and
+Initiation is a separate property rather than a fourth class. A run is either
+human-initiated or unattended, and either may occur on any class. An
+unattended run on an agentic-local surface remains agentic-local; what
+changes is that no human is present to resolve an ambiguity, which the owning
+workflow handles rather than the class.
+
+Surface class does not assign a role and does not rank surfaces. Any actor may
+occupy any role its surface class can structurally support.
+
+Executor adapters own the mapping from their concrete products onto these
+classes; this document enumerates no products. Until an adapter declares the
+class for a surface, that surface has no class under this vocabulary and this
+section places no obligation on it. Where an adapter has declared a class and
+a required contract depends on a capability that class lacks, fail closed and
 report the capability gap rather than substituting a weaker route.
 
 ## Kickoff Mutation Boundaries

--- a/scripts/claude-review
+++ b/scripts/claude-review
@@ -1683,11 +1683,40 @@ def lock_path(relative: str) -> bool:
 # repository when any actor runs an ordinary Git command there. The reviewer
 # cannot prevent it: auto-maintenance runs under the acting process, not the
 # observing one, and an unrelated worktree's operator does not share the
-# reviewer's environment. Admitting it is therefore required for the
-# unrelated-worktree concurrency guarantee, and it is admitted only as an exact
-# path and only while protected candidate evidence is unchanged. Every other
-# lock, including any other path under objects/, remains fail-closed.
+# reviewer's environment.
+#
+# The admission is deliberately narrower than "this path is tolerated". Four
+# conditions must hold together, and each closes a distinct masking route:
+#
+#   1. Live monitoring only. Terminal postflight remains fully fail-closed, so
+#      a lock that survives the attempt is still blocking. This preserves the
+#      external-ai-reviewer contract, which permits a live-monitor exception
+#      and prohibits extending it to terminal postflight.
+#   2. Creation only. A removal, replacement, content change, or mode change to
+#      this path is a mutation of pre-existing state and stays blocking.
+#   3. Regular file or already vanished. A symlink or directory at this path is
+#      never admitted, so it cannot be used to reach outside the object store.
+#   4. Protected candidate evidence unchanged, as for every other admission.
+#
+# Every other lock, including any other path under objects/, remains
+# fail-closed in both live monitoring and terminal postflight.
 GIT_BACKGROUND_MAINTENANCE_LOCKS = frozenset({"objects/maintenance.lock"})
+
+
+def admissible_background_maintenance_lock(
+    relative: str,
+    before: object | None,
+    after: object | None,
+) -> bool:
+    """Whether one observed change is a bare creation of a known Git maintenance lock."""
+
+    if relative not in GIT_BACKGROUND_MAINTENANCE_LOCKS:
+        return False
+    if before is not None:
+        return False
+    if not isinstance(after, dict):
+        return False
+    return after.get("kind") in {"file", "vanished_during_snapshot"}
 
 
 def object_storage_path(relative: str) -> bool:
@@ -1731,7 +1760,12 @@ def object_storage_layout_change(relative: str, before: object | None, after: ob
     )
 
 
-def classify_git_admin_changes(before: dict[str, object], after: dict[str, object]) -> list[dict[str, object]]:
+def classify_git_admin_changes(
+    before: dict[str, object],
+    after: dict[str, object],
+    *,
+    live_monitoring: bool = False,
+) -> list[dict[str, object]]:
     protected_evidence_unchanged = (
         before.get("head") == after.get("head")
         and before.get("protected_revisions") == after.get("protected_revisions")
@@ -1911,7 +1945,8 @@ def classify_git_admin_changes(before: dict[str, object], after: dict[str, objec
                         "protected_candidate_evidence_unchanged": True,
                     }
                 elif (
-                    relative in GIT_BACKGROUND_MAINTENANCE_LOCKS
+                    live_monitoring
+                    and admissible_background_maintenance_lock(relative, old_identity, new_identity)
                     and protected_evidence_unchanged
                 ):
                     scope = "git_background_maintenance"
@@ -1919,6 +1954,9 @@ def classify_git_admin_changes(before: dict[str, object], after: dict[str, objec
                     disposition = "tolerated"
                     evidence = {
                         "admitted_git_managed_path": relative,
+                        "admitted_observation_scope": "live_monitoring_only",
+                        "admitted_change_type": "added",
+                        "observed_kind": new_identity.get("kind") if isinstance(new_identity, dict) else None,
                         "protected_candidate_evidence_unchanged": True,
                     }
                 else:
@@ -2027,7 +2065,12 @@ def classify_git_admin_changes(before: dict[str, object], after: dict[str, objec
     return records
 
 
-def snapshot_comparison(before: dict[str, object], after: dict[str, object]) -> dict[str, object]:
+def snapshot_comparison(
+    before: dict[str, object],
+    after: dict[str, object],
+    *,
+    live_monitoring: bool = False,
+) -> dict[str, object]:
     before_body = before["body"]
     after_body = after["body"]
     blocking_paths: list[str] = []
@@ -2046,7 +2089,11 @@ def snapshot_comparison(before: dict[str, object], after: dict[str, object]) -> 
     if before_body["git_index"] != after_body["git_index"]:
         blocking_paths.append("git-index")
         raw_paths.append("git-index")
-    candidate_admin_changes = classify_git_admin_changes(before_body["git_admin"], after_body["git_admin"])
+    candidate_admin_changes = classify_git_admin_changes(
+        before_body["git_admin"],
+        after_body["git_admin"],
+        live_monitoring=live_monitoring,
+    )
     git_admin_changes.extend(candidate_admin_changes)
     before_repositories = before_body.get("guarded_git_repositories", {})
     after_repositories = after_body.get("guarded_git_repositories", {})
@@ -3182,7 +3229,9 @@ def run_governed_review(
                         )
                     else:
                         assert live_snapshot is not None
-                        live_comparison = snapshot_comparison(attempt_snapshot, live_snapshot)
+                        live_comparison = snapshot_comparison(
+                            attempt_snapshot, live_snapshot, live_monitoring=True
+                        )
                         live_changes = list(live_comparison["blocking_paths"])
                         live_observed_raw_paths.update(live_comparison["raw_changed_paths"])
                         live_observed_tolerated_paths.update(live_comparison["tolerated_paths"])
@@ -3267,7 +3316,9 @@ def run_governed_review(
                         )
                     else:
                         assert live_snapshot is not None
-                        live_comparison = snapshot_comparison(attempt_snapshot, live_snapshot)
+                        live_comparison = snapshot_comparison(
+                            attempt_snapshot, live_snapshot, live_monitoring=True
+                        )
                         live_changes = list(live_comparison["blocking_paths"])
                         live_observed_raw_paths.update(live_comparison["raw_changed_paths"])
                         live_observed_tolerated_paths.update(live_comparison["tolerated_paths"])

--- a/scripts/claude-review
+++ b/scripts/claude-review
@@ -1679,6 +1679,17 @@ def lock_path(relative: str) -> bool:
     return any(part.endswith(".lock") for part in Path(relative).parts)
 
 
+# Git's own background auto-maintenance creates this lock in the candidate
+# repository when any actor runs an ordinary Git command there. The reviewer
+# cannot prevent it: auto-maintenance runs under the acting process, not the
+# observing one, and an unrelated worktree's operator does not share the
+# reviewer's environment. Admitting it is therefore required for the
+# unrelated-worktree concurrency guarantee, and it is admitted only as an exact
+# path and only while protected candidate evidence is unchanged. Every other
+# lock, including any other path under objects/, remains fail-closed.
+GIT_BACKGROUND_MAINTENANCE_LOCKS = frozenset({"objects/maintenance.lock"})
+
+
 def object_storage_path(relative: str) -> bool:
     if re.fullmatch(r"objects/[0-9a-f]{2}", relative):
         return True
@@ -1897,6 +1908,17 @@ def classify_git_admin_changes(before: dict[str, object], after: dict[str, objec
                     evidence = {
                         "ref": unlocked_ref,
                         "provisional_attribution": "unrelated_ref_transition_not_yet_observed",
+                        "protected_candidate_evidence_unchanged": True,
+                    }
+                elif (
+                    relative in GIT_BACKGROUND_MAINTENANCE_LOCKS
+                    and protected_evidence_unchanged
+                ):
+                    scope = "git_background_maintenance"
+                    classification = "tolerated_git_background_maintenance_lock"
+                    disposition = "tolerated"
+                    evidence = {
+                        "admitted_git_managed_path": relative,
                         "protected_candidate_evidence_unchanged": True,
                     }
                 else:

--- a/tests/test_cak_local_asset_path_ratchet.py
+++ b/tests/test_cak_local_asset_path_ratchet.py
@@ -3,11 +3,38 @@
 
 from __future__ import annotations
 
+import subprocess
 import unittest
 from pathlib import Path
 
 
 ROOT = Path(__file__).resolve().parents[1]
+
+
+def tracked_files(search_root: Path) -> list[Path]:
+    """Return files Git tracks under ``search_root``.
+
+    The ratchet governs the repository's committed active surfaces. Walking the
+    filesystem instead would also read untracked local artifacts — editor state,
+    build output, or OS metadata such as ``.DS_Store`` — which are not part of
+    those surfaces and are not necessarily valid UTF-8.
+    """
+
+    result = subprocess.run(
+        [
+            "git",
+            "-C",
+            str(ROOT),
+            "ls-files",
+            "-z",
+            "--",
+            search_root.relative_to(ROOT).as_posix(),
+        ],
+        check=True,
+        stdout=subprocess.PIPE,
+        text=True,
+    )
+    return [ROOT / entry for entry in result.stdout.split("\0") if entry]
 
 # These are the reviewed active references at the CAK-178 implementation
 # baseline. New entries require an explicit ownership review rather than
@@ -51,7 +78,7 @@ class CakLocalAssetPathRatchetTests(unittest.TestCase):
         observed: set[tuple[str, str]] = set()
 
         for search_root in ACTIVE_SURFACE_ROOTS:
-            for path in search_root.rglob("*"):
+            for path in tracked_files(search_root):
                 if not path.is_file() or "__pycache__" in path.parts:
                     continue
                 relative = path.relative_to(ROOT).as_posix()

--- a/tests/test_claude_review_launcher.py
+++ b/tests/test_claude_review_launcher.py
@@ -3872,6 +3872,47 @@ class GovernedClaudeReviewLauncherTests(unittest.TestCase):
         self.assertEqual(symlink_record["change_type"], "added")
         self.assertEqual(symlink_record["classification"], "blocking_lock_change")
 
+    def test_git_background_maintenance_lock_is_tolerated_and_others_stay_blocking(self):
+        module = load_script("claude_review_maintenance_lock", LAUNCHER)
+        environment = os.environ.copy()
+        git_directory = self.candidate / ".git"
+        objects_directory = git_directory / "objects"
+        objects_directory.mkdir(parents=True, exist_ok=True)
+
+        maintenance_lock = objects_directory / "maintenance.lock"
+        baseline = module.source_snapshot([self.candidate], self.candidate, environment)
+        maintenance_lock.write_text("", encoding="utf-8")
+        changed = module.source_snapshot([self.candidate], self.candidate, environment)
+        comparison = module.snapshot_comparison(baseline, changed)
+        record = next(
+            record
+            for record in comparison["git_admin_changes"]
+            if record["path"] == "objects/maintenance.lock"
+        )
+        self.assertEqual(record["classification"], "tolerated_git_background_maintenance_lock")
+        self.assertEqual(record["disposition"], "tolerated")
+        self.assertTrue(record["evidence"]["protected_candidate_evidence_unchanged"])
+        self.assertTrue(comparison["passed"])
+        maintenance_lock.unlink()
+
+        # The admission is exact. A neighbouring lock under the same directory
+        # must remain fail-closed so this does not become a broad objects/
+        # exclusion.
+        neighbour_lock = objects_directory / "maintenance-other.lock"
+        neighbour_baseline = module.source_snapshot([self.candidate], self.candidate, environment)
+        neighbour_lock.write_text("", encoding="utf-8")
+        neighbour_changed = module.source_snapshot([self.candidate], self.candidate, environment)
+        neighbour_comparison = module.snapshot_comparison(neighbour_baseline, neighbour_changed)
+        neighbour_record = next(
+            record
+            for record in neighbour_comparison["git_admin_changes"]
+            if record["path"] == "objects/maintenance-other.lock"
+        )
+        self.assertEqual(neighbour_record["classification"], "blocking_lock_change")
+        self.assertEqual(neighbour_record["disposition"], "blocking")
+        self.assertFalse(neighbour_comparison["passed"])
+        neighbour_lock.unlink()
+
     def test_linked_worktree_common_git_admin_locks_are_detected(self):
         module = load_script("claude_review_linked_worktree_locks", LAUNCHER)
         environment = os.environ.copy()

--- a/tests/test_claude_review_launcher.py
+++ b/tests/test_claude_review_launcher.py
@@ -3872,28 +3872,85 @@ class GovernedClaudeReviewLauncherTests(unittest.TestCase):
         self.assertEqual(symlink_record["change_type"], "added")
         self.assertEqual(symlink_record["classification"], "blocking_lock_change")
 
-    def test_git_background_maintenance_lock_is_tolerated_and_others_stay_blocking(self):
-        module = load_script("claude_review_maintenance_lock", LAUNCHER)
-        environment = os.environ.copy()
-        git_directory = self.candidate / ".git"
-        objects_directory = git_directory / "objects"
-        objects_directory.mkdir(parents=True, exist_ok=True)
+    def _maintenance_lock_record(self, module, environment, mutate, *, live_monitoring):
+        """Apply one mutation under .git and return its classification record."""
 
-        maintenance_lock = objects_directory / "maintenance.lock"
         baseline = module.source_snapshot([self.candidate], self.candidate, environment)
-        maintenance_lock.write_text("", encoding="utf-8")
+        mutate()
         changed = module.source_snapshot([self.candidate], self.candidate, environment)
-        comparison = module.snapshot_comparison(baseline, changed)
+        comparison = module.snapshot_comparison(baseline, changed, live_monitoring=live_monitoring)
         record = next(
             record
             for record in comparison["git_admin_changes"]
             if record["path"] == "objects/maintenance.lock"
         )
+        return comparison, record
+
+    def test_git_background_maintenance_lock_admission_is_bounded(self):
+        module = load_script("claude_review_maintenance_lock", LAUNCHER)
+        environment = os.environ.copy()
+        objects_directory = self.candidate / ".git" / "objects"
+        objects_directory.mkdir(parents=True, exist_ok=True)
+        maintenance_lock = objects_directory / "maintenance.lock"
+
+        # Admitted: bare creation of a regular file, observed live.
+        comparison, record = self._maintenance_lock_record(
+            module,
+            environment,
+            lambda: maintenance_lock.write_text("", encoding="utf-8"),
+            live_monitoring=True,
+        )
         self.assertEqual(record["classification"], "tolerated_git_background_maintenance_lock")
         self.assertEqual(record["disposition"], "tolerated")
-        self.assertTrue(record["evidence"]["protected_candidate_evidence_unchanged"])
+        self.assertEqual(record["evidence"]["admitted_observation_scope"], "live_monitoring_only")
+        self.assertEqual(record["evidence"]["observed_kind"], "file")
         self.assertTrue(comparison["passed"])
         maintenance_lock.unlink()
+
+        # Terminal postflight is unchanged by the admission. The same creation
+        # that live monitoring tolerates must still block at the terminal
+        # boundary, so a lock surviving the attempt is never masked.
+        comparison, record = self._maintenance_lock_record(
+            module,
+            environment,
+            lambda: maintenance_lock.write_text("", encoding="utf-8"),
+            live_monitoring=False,
+        )
+        self.assertEqual(record["classification"], "blocking_lock_change")
+        self.assertEqual(record["disposition"], "blocking")
+        self.assertFalse(comparison["passed"])
+
+        # Mutations of a pre-existing lock are not creations and stay blocking
+        # even under live monitoring.
+        for label, mutate in (
+            ("content_changed", lambda: maintenance_lock.write_text("altered\n", encoding="utf-8")),
+            ("mode_changed", lambda: maintenance_lock.chmod(0o600)),
+            ("removed", maintenance_lock.unlink),
+        ):
+            with self.subTest(change=label):
+                comparison, record = self._maintenance_lock_record(
+                    module, environment, mutate, live_monitoring=True
+                )
+                self.assertEqual(record["disposition"], "blocking")
+                self.assertFalse(comparison["passed"])
+
+        # A symlink or directory at the admitted path is never admitted, so the
+        # exception cannot be used to reach outside the object store.
+        for label, mutate in (
+            ("symlink", lambda: maintenance_lock.symlink_to(self.candidate / "tracked.txt")),
+            ("directory", lambda: maintenance_lock.mkdir()),
+        ):
+            with self.subTest(created=label):
+                comparison, record = self._maintenance_lock_record(
+                    module, environment, mutate, live_monitoring=True
+                )
+                self.assertEqual(record["classification"], "blocking_lock_change")
+                self.assertEqual(record["disposition"], "blocking")
+                self.assertFalse(comparison["passed"])
+                if label == "symlink":
+                    maintenance_lock.unlink()
+                else:
+                    maintenance_lock.rmdir()
 
         # The admission is exact. A neighbouring lock under the same directory
         # must remain fail-closed so this does not become a broad objects/
@@ -3902,7 +3959,9 @@ class GovernedClaudeReviewLauncherTests(unittest.TestCase):
         neighbour_baseline = module.source_snapshot([self.candidate], self.candidate, environment)
         neighbour_lock.write_text("", encoding="utf-8")
         neighbour_changed = module.source_snapshot([self.candidate], self.candidate, environment)
-        neighbour_comparison = module.snapshot_comparison(neighbour_baseline, neighbour_changed)
+        neighbour_comparison = module.snapshot_comparison(
+            neighbour_baseline, neighbour_changed, live_monitoring=True
+        )
         neighbour_record = next(
             record
             for record in neighbour_comparison["git_admin_changes"]

